featurebyte/query_graph/node/generic.py

```diff
@@ -475,7 +475,7 @@ class ForwardAggregateParameters(BaseGroupbyParameters):
     """
 
     name: str
-    horizon: Optional[str]
+    window: Optional[str]
     # TableDetails are needed because we will be performing the aggregation query directly on the calling table.
     table_details: TableDetails
```

Comment:
nit: Would be good to remove `table_details` in this or coming PR.

Reply:
will do in follow-up!